src/components/AgreementSections/CoveredEResourcesList.js

```diff
@@ -73,7 +74,14 @@ export default class CoveredEResourcesList extends React.Component {
       const titleInstance = get(e._object, 'pti.titleInstance', {});
       return getResourceIdentifier(titleInstance, 'eissn') || getResourceIdentifier(titleInstance, 'pissn');
     },
-    platform: e => get(e._object, 'pti.platform.name', '-'),
+    platform: e => {
+      return (
+        <div>
```

Comment:
Can this whole thing be a component? `<PlatformLink>` or something? You've got basically the same code copy-and-pasted three times in a single PR.

Reply:
Thought about it, but I wasnt sure if with the upcoming MCL changes we d have to re-work the whole thing..but I guess it wont affect this functionality. Will add a new component